components/content-service-api/go/blobs.pb.go

```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.1
+// 	protoc        (unknown)
```

Comment:
Could we make the protoc version not to unknow?

Reply:
Unsure. It's using the same already installed protoc, it's weird it's not reporting it into here. Will TAL.

Reply:
https://github.com/bufbuild/buf/issues/799#issuecomment-996881066